src/main/java/uk/ac/ebi/ena/submit/ContextE.java

```diff
@@ -4,25 +4,73 @@
+import uk.ac.ebi.ena.assembly.SequenceAssemblyWebinCli;
+import uk.ac.ebi.ena.assembly.TranscriptomeAssemblyWebinCli;
 import uk.ac.ebi.ena.manifest.FileFormat;
+import uk.ac.ebi.ena.rawreads.RawReadsWebinCli;
+import uk.ac.ebi.ena.webin.cli.AbstractWebinCli;
 
-public enum ContextE {
-    sequence("Sequence assembly: ASSEMBLYNAME", "SEQUENCE_FLATFILE", new FileFormat[] {FileFormat.TSV, FileFormat.FLATFILE, FileFormat.INFO} ),
-    transcriptome("Transcriptome assembly: ASSEMBLYNAME", "TRANSCRIPTOME_ASSEMBLY", new FileFormat[] {FileFormat.FASTA,FileFormat.FLATFILE, FileFormat.INFO} ),
-    genome("Genome assembly: ASSEMBLYNAME", "SEQUENCE_ASSEMBLY",new FileFormat[] {FileFormat.FASTA, FileFormat.AGP, FileFormat.FLATFILE,FileFormat.CHROMOSOME_LIST,FileFormat.UNLOCALISED_LIST,FileFormat.INFO});
-
+public enum
+ContextE
+{
+    sequence( "Sequence assembly: %s", 
+    		  "SEQUENCE_FLATFILE", 
+    		  new FileFormat[] { FileFormat.TSV, 
+    				                     FileFormat.FLATFILE, 
+    				                     FileFormat.INFO },
+    		  SequenceAssemblyWebinCli.class ),
+    
+    transcriptome( "Transcriptome assembly: %s", 
+    		       "TRANSCRIPTOME_ASSEMBLY", 
+    		       new FileFormat[] { FileFormat.FASTA,
+    		    		                      FileFormat.FLATFILE,
+    		    		                      FileFormat.INFO },
+    		       TranscriptomeAssemblyWebinCli.class ),
+    
+    genome( "Genome assembly: %s", 
+    		"SEQUENCE_ASSEMBLY",
+    		new FileFormat[] { FileFormat.FASTA, 
+    				                   FileFormat.AGP,
+    				                   FileFormat.FLATFILE,
+    				                   FileFormat.CHROMOSOME_LIST,
+    				                   FileFormat.UNLOCALISED_LIST,
+    				                   FileFormat.INFO },
+    		GenomeAssemblyWebinCli.class ),
+	reads( "Raw reads: %s",
+		   "RUN_PROCESS",
+		   new FileFormat[] {  },
+		   RawReadsWebinCli.class );
+	
     private String analysisTitle;
     private String analysisType;
     private FileFormat[] fileFormats;
+    private Class<? extends AbstractWebinCli> klass;
     
-    private ContextE(String analysisTitle, String analysisType,FileFormat[] fileFormats) {
+    private 
+    ContextE( String analysisTitle, 
+    		  String analysisType, 
+    		  FileFormat[] fileFormats,
+    		  Class<? extends AbstractWebinCli> klass ) 
+    {
         this.analysisTitle = analysisTitle;
-        this.analysisType = analysisType;
-        this.fileFormats =fileFormats;
+        this.analysisType  = analysisType;
+        this.fileFormats   = fileFormats;
+        this.klass         = klass;
     }
 
-    public String getAnalysisTitle(String name) {
-        return analysisTitle.replace("_NAME_", name);
+    
+    public Class<? extends AbstractWebinCli>
+    getValidatorClass()
+    {
+    	return this.klass;
+    }
+
+    
+    public String 
+    getAnalysisTitle( String name )
+    {
+        return analysisTitle.replace( "ASSEMBLYNAME", name );
```

Comment:
- Change method name to getTitle.
- Change analysiTitle to title
- Add unit test. Add test class ContenxtETest with test methods testTitle to make sure that title is correct for each enum.